### PR TITLE
Add legacytools.hpa.org.uk as an alias of phe_hpa

### PR DESCRIPTION
**Pull request by issyl0**
- This was suggested as a good approach in
  https://trello.com/c/psVtDaSe/71-configure-legacytools-hpa-org-uk-in-the-transition-tool.

### CODE REVIEW

File: data/transition-sites/phe_hpa.yml [hunk 1]
```diff
@@ -9,3 +9,4 @@ aliases:
 - hpa.org.uk
 - www.hpa.nhs.uk
 - hpa.nhs.uk
+- legacytools.hpa.org.uk
```
Comment (jennyd) [inline data/transition-sites/phe_hpa.yml]: Does this domain serve the original content at the same paths as it was on www.hpa.org.uk, or was the content moved to different paths for the legacytools domain? It only makes sense to add this domain as an alias if the same mappings can be used for both. /cc @marksheldonGDS

**Comment by issyl0** [comment]
This is blocked pending comments from @marksheldonGDS (I can't work out whether paths are the same or not), so I'm closing this in the meantime on the advice of @dsingleton.

File: data/transition-sites/phe_hpa.yml (hunk 1 above)
Comment (marksheldonGDS) [inline data/transition-sites/phe_hpa.yml]: No, it doesn't. So this should be setup as a configured site for Public Health England. Apologies for the lack of detail.

Comment (issyl0) [inline data/transition-sites/phe_hpa.yml]: OK, thanks. I'll change it around and make it a new PR.

**Comment by issyl0** [comment]
Resolved. This is the correct approach now, apparently, instead of #1121.

**Pull request merged**